tests/e2e_operations.py

```diff
@@ -87,31 +108,65 @@ def test_forced_ledger_chunk(network, args):
     # Check that there is indeed a ledger chunk that ends at the
     # first signature after proposal.completed_seqno
     ledger = ccf.ledger.Ledger(ledger_dirs)
```

Comment:
This ignores uncommitted ledger files by default. How do we guarantee that the proposal completion is in a committed chunk here?